### DISCUSSION TITLE
Is repo maintained by somebody? 119 issues, 31 PR unresolved ...

### DISCUSSION THREAD

**Discussion by vsfedorenko** [dotansimha/graphql-code-generator#4027] [General] [answered]
Hi, 

What's wrong? Why don't you resolve all of this stuff?

**Comment by vsfedorenko** [comment]
I've posted an issue, fixed it and covered with tests
https://github.com/dotansimha/graphql-code-generator/issues/3998

And after almost a week after this NOBODY has responded to me. 
Why? Is it really repo with ~5k stars and ~3.5k usages?

**Comment by dotansimha** [comment, marked as answer]
Hi @meiblorn,

This repo is maintained by me, @ardatan, @kamilkisiela and @Urigo (The Guild). We recently published [a new version with some fixes](https://github.com/dotansimha/graphql-code-generator/releases/tag/v1.13.5), we got a [new website with **live** support chat](https://graphql-code-generator.com/). Have you seen recently an open-source project that provides a live support chat (with a 1h average response time), for free? 

We are constantly trying to improve the codegen and [our other open-source projects](https://github.com/the-guild-org/Stack), and we've also [started to collect issues and ideas for v2](https://github.com/dotansimha/graphql-code-generator/discussions/3955). 

We are trying to keep up with everything going on, and sometimes we miss some issues or PRs. It happens. Sorry if we missed your PR, this was not intentional, we just had a busy week 😊 I will find some time this week to go over your PR (and others), and hopefully to release a new version of codegen, including [our latest work for GraphQL-Modules and GraphQL-Codegen integration](https://github.com/dotansimha/graphql-code-generator/pull/3923) and [GraphQL-Tools v6 support](https://github.com/dotansimha/graphql-code-generator/pull/4019).
